modules/sno-du-managing-a-cluster-configuration.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-managing-a-cluster-configuration_{context}"]
+= Managing a cluster configuration
+
```

Comment:
The ManagedCluster CR is part of enrollment of the installed cluster with the RHACM hub cluster.